### ISSUE TITLE
issue of show_image declaration 

### ISSUE THREAD

**Issue by lianghong** [madhawav/YOLO3-4-Py#29]
when i build project with the command "python3 setup.py  install", I have following error

pydarknet.cpp:2164:46: error: too few arguments to function ‘int show_image(image, const char*, int)’
   show_image(__pyx_v_self->img, __pyx_v_title);
![screenshot from 2018-08-27 08-02-16](https://user-images.githubusercontent.com/3388147/44634690-8d5ff280-a9cf-11e8-97a8-b9f8f0e5458e.png)

BTW, my OS is Ubuntu 18.04

**Comment by madhawav** [comment]
Duplicate of #28. Fixed with #30

**Comment by josefernandez91** [comment]
Dear @madhawav ,
I just downloaded the branch master where it has the applied fix that they describe, but I get the same error. I leave error capture.

python3 setup.py build_ext --inplace


creating build/temp.linux-x86_64-3.6
x86_64-linux-gnu-gcc -pthread -DNDEBUG -g -fwrapv -O2 -Wall -g -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -DUSE_GPU=0 -DUSE_CV=1 -I/home/everis/.local/lib/python3.6/site-packages/numpy/core/include -I/home/everis/Documentos/darknet/include -I/home/everis/Documentos/darknet/src -I/usr/include/python3.6m -c pydarknet.cpp -o build/temp.linux-x86_64-3.6/pydarknet.o -I/usr/include/python3.6m -I/usr/include/x86_64-linux-gnu/python3.6m -I/usr/local/include/opencv -I/usr/local/include
In file included from /home/everis/.local/lib/python3.6/site-packages/numpy/core/include/numpy/ndarraytypes.h:1832:0,
                 from /home/everis/.local/lib/python3.6/site-packages/numpy/core/include/numpy/ndarrayobject.h:12,
                 from /home/everis/.local/lib/python3.6/site-packages/numpy/core/include/numpy/arrayobject.h:4,
                 from pydarknet.cpp:612:
/home/everis/.local/lib/python3.6/site-packages/numpy/core/include/numpy/npy_1_7_deprecated_api.h:17:2: warning: #warning "Using deprecated NumPy API, disable it with " "#define NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION" [-Wcpp]
 #warning "Using deprecated NumPy API, disable it with " \
  ^~~~~~~
**pydarknet.cpp: In function ‘PyObject* __pyx_pf_9pydarknet_5Image_2show_image(__pyx_obj_9pydarknet_Image*, char*, int)’:
pydarknet.cpp:2315:75: error: too many arguments to function ‘void show_image(image, const char*)’
   show_image(__pyx_v_self->img, __pyx_v_title, __pyx_v_wait_duration_in_ms);**
                                                                           ^
In file included from bridge.h:16:0,
                 from pydarknet.cpp:618:
/home/everis/Documentos/darknet/src/image.h:69:6: note: declared here
 void show_image(image p, const char *name);
      ^~~~~~~~~~
**pydarknet.cpp: In function ‘PyObject* __pyx_pf_9pydarknet_8Detector_4detect(__pyx_obj_9pydarknet_Detector*, __pyx_obj_9pydarknet_Image*, float, float, float)’:
pydarknet.cpp:3060:163: error: too few arguments to function ‘detection* get_network_boxes(network*, int, int, float, float, int*, int, int*, int)’
   __pyx_v_dets = get_network_boxes(__pyx_v_self->net, __pyx_v_image->img.w, __pyx_v_image->img.h, __pyx_v_thresh, __pyx_v_hier_thresh, ((int *)0), 0, __pyx_v_pnum);**
                                                                                                                                                                   ^
In file included from bridge.h:15:0,
                 from pydarknet.cpp:618:
/home/everis/Documentos/darknet/include/darknet.h:903:20: note: declared here
 LIB_API detection *get_network_boxes(network *net, int w, int h, float thresh, float hier, int *map, int relative, int *num, int letter);
                    ^~~~~~~~~~~~~~~~~
pydarknet.cpp: In function ‘void __pyx_pf_9pydarknet_8Detector_6__dealloc__(__pyx_obj_9pydarknet_Detector*)’:
pydarknet.cpp:3312:30: error: could not convert ‘__pyx_v_self->__pyx_obj_9pydarknet_Detector::net’ from ‘network*’ to ‘network’
   free_network(__pyx_v_self->net);
                ~~~~~~~~~~~~~~^~~
**error: command 'x86_64-linux-gnu-gcc' failed with exit status 1**
